web/controllers/user_controller.ex

```diff
@@ -13,6 +13,10 @@ defmodule ResearchResource.UserController do
     changeset = User.registration_changeset(%User{}, user_params)
     case Repo.insert(changeset) do
       {:ok, user} ->
+        subject = "Welcome to Research Resource"
+        message = "Welcome to Research Resource, #{user.first_name}"
+        ResearchResource.Email.send_email(user.email, subject, message)
```

Comment:
You could consider doing this asynchronously using `Task.async`: https://github.com/InclusiveClassrooms/skills-wheel/blob/9cba045985aa44ecb7411aab7a94c9e33537c8a0/web/controllers/forgotpass_controller.ex#L57